<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogNewProject</class>
 <widget class="QDialog" name="DialogNewProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1043</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New TRSE project</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Select a project template</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="1,10,4">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,15">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Select system:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbSystem">
          <property name="font">
           <font>
            <pointsize>16</pointsize>
           </font>
          </property>
          <item>
           <property name="text">
            <string>C64</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>VIC20</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>C128</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PET</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>NES</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SNES</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>GAMEBOY</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PLUS4</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>SPECTRUM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TRS80COCO</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>X86</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>OK64</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AMIGA</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ATARI2600</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TVC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>THOMSON</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ATARI520ST</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AMSTRADCPC</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>BBCM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MEGA65</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>TIM</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>ATARI800</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>VZ200</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" rowspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="5,5">
        <item>
         <widget class="QListWidget" name="lstData"/>
        </item>
        <item>
         <widget class="QTextBrowser" name="txtInfo"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Project options</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="leProjectName">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Project name:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="leProjectDir"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Project directory</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="btnSelectDir">
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblInfo">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOk">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cmbSystem</tabstop>
  <tabstop>txtInfo</tabstop>
  <tabstop>btnSelectDir</tabstop>
  <tabstop>leProjectName</tabstop>
  <tabstop>leProjectDir</tabstop>
  <tabstop>btnOk</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>lstData</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
